diff -ruN xf86-input-cmt/src/gesture.c xf86-input-cmt.new/src/gesture.c
--- xf86-input-cmt/src/gesture.c	2015-07-26 19:36:21.869483673 -0600
+++ xf86-input-cmt.new/src/gesture.c	2015-07-26 19:36:55.146093175 -0600
@@ -140,8 +140,8 @@
     hwprops.bottom          = props->area_bottom;
     hwprops.res_x           = props->res_x;
     hwprops.res_y           = props->res_y;
-    hwprops.screen_x_dpi    = 133;
-    hwprops.screen_y_dpi    = 133;
+    hwprops.screen_x_dpi    = 96;
+    hwprops.screen_y_dpi    = 96;
     hwprops.orientation_minimum = props->orientation_minimum;
     hwprops.orientation_maximum = props->orientation_maximum;
     hwprops.max_finger_cnt  = evstate->slot_count;
